pyeasee/easee.py

```diff
@@ -182,6 +198,146 @@ async def close(self):
             await self.session.close()
             self.session = None
 
+        await self._sr_disconnect()
+
+    def _sr_token(self):
+        """
+        Return access token to signalr library, called from signalr thread, internal use only
+        """
+        if self.running_loop is not None:
+            future = asyncio.run_coroutine_threadsafe(self._verify_updated_token(), self.event_loop)
+            future.result()
```

Comment:
Should we wait for the future to be done?

Reply:
future.result() does actually wait until the function is done..
(This is called from the the signalrcore synchronous lib thread so we can not use await).
See https://docs.python.org/3/library/concurrent.futures.html#concurrent.futures.Future